scripts/commands/update.py

```diff
@@ -6,45 +6,25 @@
 from shared import spreadsheet_odoo_versions
 from utils import pushd, get_o_spreadsheet_hash
 from helpers import (
-    get_existing_prs,
     checkout,
-    reset,
-    commit_message,
-    run_dist,
-    copy_dist,
+    get_commits,
+    copy_build,
+    odoo_commit_title,
+    fetch_repositories,
+    get_odoo_prs,
     make_PR,
+    reset,
+    run_build,
 )
 
 
 def update(config: configparser.ConfigParser):
-    print("\n=== UPDATE ===\nThis may take a while ;-)\n")
+    print("\n=== UPDATE ODOO ===\nThis may take a while ;-)\n")
```

Comment:
shouldn't it nous envoyer chier if the commit isn't a `[REL]` commit?

Reply:
Done afterwards in the function :)